### ISSUE TITLE
--verbose and --timestamp options are not considered in toml config file 

### ISSUE THREAD

**Issue by oPromessa** [RhetTbull/osxphotos#643]
**Describe the bug**
`osxphotos 0.47.1` **does** recognise `--verbose` and `--timestamp` options as arguments, it does not seem to take them in consideration when read from the `.toml` config file with the `--load-config` option.

**To Reproduce**
1. What' the full command line you used with osxphotos?
* I've used this command line to setup `new.toml` file:
```sh
$ rm -fr ola; mkdir ola; osxphotos export ola --exiftool --uuid $pic --update --cleanup --exiftool-merge-keywords --exiftool-merge-persons --keyword-template "{keyword}" --keyword-template "{label}" --touch-file --strip --skip-original-if-edited --timestamp --verbose --retry 2 --description-template  "Album:{album,}{newline}Description:{descr,}{newline}Person:{person,}" --not-hidden --directory "{album[/,.|:,.]}" --touch-file --save-config new.toml 
2022-02-27 17:01:08.289424 -- osxphotos version 0.47.1
2022-02-27 17:01:08.289505 -- Saving options to config file 'new.toml'
2022-02-27 17:01:08.290145 -- exiftool path: /usr/local/bin/exiftool
Using last opened Photos library: /Users/Shared/Pictures/iPhoto Shared Library.photoslibrary
(...)
```
* which produced the `new.toml` file
```toml
[export]
cleanup = true
description_template = "Album:{album,}{newline}Description:{descr,}{newline}Person:{person,}"
directory = "{album[/,.|:,.]}"
exiftool = true
exiftool_merge_keywords = true
exiftool_merge_persons = true
keyword_template = [ "{keyword}", "{label}",]
not_hidden = true
retry = 2
skip_original_if_edited = true
strip = true
timestamp = true
touch_file = true
update = true
uuid = [ "EAA15CD0-4606-42D8-9944-52CDDFD1DA62",]
verbose = true
```
2. What was the error output?
* when I used it with the `--loadconfig new.toml` file the output is not verbose and does not contain the timestamp.
```sh
$ rm -fr ola; mkdir ola; osxphotos export ola  --load-config new.toml 
Using last opened Photos library: /Users/Shared/Pictures/iPhoto Shared Library.photoslibrary
Exporting 1 photo to /Users/MSP/Documents/GitHub/osxphotos/ola...
Processed: 1 photo, exported: 1, updated: 0, skipped: 0, updated EXIF data: 1, missing: 0, error: 0, touched date: 1
Elapsed time: 0:00:00
Cleaning up /Users/MSP/Documents/GitHub/osxphotos/ola
Deleted: 0 files, 0 directories
```

**Expected behavior**
Would expect the `--verbose`and `--timestamp`options to kick-in.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: [e.g. which version of macOS]
```
ProductName:	macOS
ProductVersion:	12.2.1
BuildVersion:	21D62
```
 - osxphotos version (`osxphotos --version`): `osxphotos, version 0.47.1`

**Additional context**
Running it from the repository clone installed for testing.

**Comment by RhetTbull** [comment]
Fixed in [v0.47.2](https://github.com/RhetTbull/osxphotos/releases/tag/v0.47.2)

**Comment by RhetTbull** [comment]
I'd done some big refactoring which included rewriting the verbose code which had used a global variable (and was causing problems for the test suite) to use a factory method and closure instead of global.
